awx/ui_next/src/screens/Job/JobList/JobList.jsx

```diff
@@ -160,7 +160,7 @@ class JobList extends Component {
             hasContentLoading={hasContentLoading}
             items={jobs}
             itemCount={itemCount}
-            itemName={itemName}
+            itemName={itemCount === 1 ? i18n._(t`Job`): i18n._(t`Jobs`)}
```

Comment:
We can't translate these here, because it needs to go inside a full sentence message inside PDL; then the full sentence gets translated as a whole by PDL. Same with other List components below

Reply:
Good catch here!